lom/src/lib/lib_test/config/tx2_test.go

```diff
@@ -233,9 +232,9 @@ func TestGoroutineTracker(t *testing.T) {
         var names string
         for _, info := range infos {
             if gi, ok := info.(lomcommon.GoroutineInfo); ok {
-                if gi.Status == lomcommon.GoroutineStatusRunning {
-                    names = gi.Name
-                }
+
```

Comment:
A nice candidate for table driven approach. You would find better one. Here is a raw thinking...
test_entry = {
     start_list:  [list of routines to start -0, 1 , 2...]
     exp_running: [ ...]
     exp_not_runnig: [ ...]
}

TestTable = {
{  [0,1, 2], [0, 1, 2], [3, 4, 5, 6] },
{ [4,5], [0,1,2,4, 5], [3,6] },
{ [], [0,1,2,4,5],[3,6]},
...
}

Loop it.
Anytime copy/paste -- Think about looping 

We are going to track how much of our test code is data based and publish it.

Reply:
Sure. Will add in future as it need a lot of refactoring of unit tests.